### PR TITLE
GNOME - Use docs helper

### PR DESCRIPTION
**Pull request by fsobolev**

### CODE REVIEW

File: NickvisionTubeConverter.GNOME/Helpers/Help.cs [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    public static string GetHelpURL(string pageName)
+    {
+        if (string.IsNullOrEmpty(Environment.GetEnvironmentVariable("SNAP")))
+        {
+            return $"help:parabolic/{pageName}";
+        }
+        var lang = "C";
+        if (!CultureInfo.CurrentCulture.Equals(CultureInfo.InvariantCulture) && CultureInfo.CurrentCulture.Name != "en-US")
+        {
+            using var linguasStream = Assembly.GetCallingAssembly().GetManifestResourceStream("NickvisionTagger.GNOME.LINGUAS");
```
Comment (nlogozzo) [inline NickvisionTubeConverter.GNOME/Helpers/Help.cs:28]: TubeConverter

**Review by nlogozzo — CHANGES_REQUESTED** [review]

File: NickvisionTubeConverter.GNOME/Helpers/Help.cs (hunk 1 above)
Comment (fsobolev) [reply NickvisionTubeConverter.GNOME/Helpers/Help.cs:28]: I can't believe I missed this haha

Comment (nlogozzo) [reply NickvisionTubeConverter.GNOME/Helpers/Help.cs]: The copy and paste is strong 😅

**Review by nlogozzo — APPROVED** [review]

**Comment by nlogozzo** [comment]
https://github.com/NickvisionApps/Parabolic/releases/tag/2023.8.0-rc1

**Pull request merged**